bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts

```diff
@@ -27,7 +28,8 @@ export class ContactComponent implements OnInit {
   constructor(private quoteService: QuoteService,
               private spinner: NgxSpinnerService,
               private appAlertOverlayModalService: AppAlertOverlayModalService,
-              public scrollTo: ScrollService) {
+              public scrollTo: ScrollService,
```

Comment:
Make constructor variables private.  Rename `scrollTo` variable to `scrollToService`.

Reply:
renamed to scrollToService and adjusted code for it.